src/main.js

```diff
@@ -4,14 +4,82 @@ import { spawn } from 'child_process'
 import parser from 'xml2json'
 import openAboutWindow from 'about-window'
 
-import browsers from './browsers'
+// import browsers from './browsers'
+
+import memFs from 'mem-fs'
+import editor from 'mem-fs-editor'
 
 // Keep a global reference of the window object, if you don't, the window will
 // be closed automatically when the JavaScript object is garbage collected.
 let pickerWindow = null
 let tray = null
 let appIsReady = false
 
+let configFileName = 'browserosaurus.json'
+
+var browsers = {}
+var config = {}
+var version = app.getVersion()
+
+const loadConfig = () => {
+  return new Promise((fulfill, reject) => {
+    var notifications = []
+
+    var configPath = require('os').homedir() + '/' + configFileName
+    var configLocalPath = './src/browserosaurus.json'
```

Comment:
could we make a dot file? I think config files should really be dot files. What do you reckon?

Reply:
A lot of config files reside in `~/.config/`, perhaps that would be a good place to store these files? Perhaps `~/.config/browsersaurus/`.

Reply:
Happy to go with `~/.config` if that's the consensus. However I really don't think this is ever going to be more complicated that one json file, so howabout `~/.config/.browserosaurus.json` ?

Reply:
`~/.config/browserosaurus.json`? I don't think there's any need for files within `~/.config` to be hidden.

Reply:
Yeah, I realised as I replied. I’m happy with that. Let’s see what Jackymancs4 reckons.

Reply:
Sorry for being late, very busy days. 
As said in the side notes, the file is not a dot file just for testing purpose.. will be in the final PR. For that, changing line 18 in file main.js will do the trick. 

So changing
```
let configFileName = 'browserosaurus.json'
```
to 
```
let configFileName = '.browserosaurus.json'
```
or
```
let configFileName = '.config/browserosaurus.json'
```
Is just a matter of preferences.
The only problem that rise (and it's not a real problem) is that right now I don't know if the library I'm using will implicitly create non existing folders in case of 
```
let configFileName = '.config/browserosaurus/config.json'
```
Needs investigation.
For what concern my preference, I think I like ```'.config/browserosaurus.json'``` the most.

Reply:
Thanks. Ok, agreed on `.config/browserosaurus.json`.